### ISSUE TITLE
Led component

### ISSUE THREAD

**Issue by lastmjs** [scramjs/jfive-web-components#1]

**Comment by bdemann** [comment]
I was reading through the LED documentation and it says that the pin value can be "a Number or String address for the Led pin (digital/PWM)." If it takes either a number or a string we shouldn't have to check what the user is giving us we can just put it into the jfive API? @lastmjs or should we be checking that input? Or any of the input? In other words how much responsibility is on us for correct input into the attributes?

**Comment by lastmjs** [comment]
I don't think we should check the input, just pass it along. If they mess it up it will break, but they need to ensure they don't do that. It's the defensive programming versus not debate, and I think we shouldn't be too defensive in this case, just to keep it simple
